### ISSUE TITLE
Copy/paste bug

### ISSUE THREAD

**Issue by AGonLen** [Thomas101/wmail#522]
I'm using WMail 2.0.9. The issue is with the copy (CTRL+C)/paste (CTRL+V).

I'll try explain:
You have an email written with a few lines, you double click 1 word (and it selects the word + 1 space after the word), then you click anywhere in your text and paste it, works fine as pastes the word, but then the program highlights the rest of the text in the paragraph (and you can accidentally delete the email)

BTW, have to say that this issue does not happen if you select the exact word, without the space at the end, nor when you select multiple words but without the extra space after the last word.

Hope this helps, but happy to provide more info if needed.

**Comment by gajamukhu** [comment]
+1

This is a serious problem with the current stable release 2.0.0, as well. When I try to select a sentence or a paragraph with my mouse, most of the text on the screen gets selected, which renders futile the original attempt to select the text. However, if I first select a word in the desired sentence/paragraph by double-clicking it and then use the keyboard to select the rest of the text, there are no issues. While this is a viable workaround, it is an awkward one.

**Comment by Thomas101** [comment]
Hi, I've given this a try and wasn't able to see the behaviour you described. Which operating system are you using? Is this issue with gmail/google inbox/both?

**Comment by gajamukhu** [comment]
Hi Thomas,

Thanks for your attention to this. I'm using Gmail only (not inbox) on
Windows 10. Hope that helps.

Below is a screenshot of my attempt of using my mouse to select the first
sentence of your email, beginning with "Hi, ...". You can see how it
selected everything before the "Hi, ..." and even selected material on the
parallel pane.

Ganesh

[image: Inline image 2]

**Comment by AGonLen** [comment]
Me too, I'm using WMail 2.0.9 on Windows 10 and GMail only, and have the same exact issue as described by @gajamukhu

**Comment by Thomas101** [comment]
Thanks! If you use the 2.0.9 prerelease and go to Settings, then the Advanced tab and Toggle Use Zoom for DSF to be off does this make a difference?

**Comment by AGonLen** [comment]
Nope. I changed the setting in the Advanced tab and the same thing happened.

I recorded a video with my phone to show it to you: h**p://sendvid.com/5a571rvy

**Comment by Thomas101** [comment]
Thanks for the video, that's really helpful. I suspect this is a bug with chromium on windows but I'll look into it further. There is a new beta version of chromium so once I've integrated that into WMail I'll take another look. Hopefully it will fix this behaviour.

**Comment by AGonLen** [comment]
Yep, hopefully!!! Thanks for looking into it.

**Comment by gajamukhu** [comment]
Yes, thanks for looking into it, Thomas.
